### PR TITLE
QAE-347 - E2E test case for enable breadcrumb on various page for after header position

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for enable breadcrumb on various page for inside header position

### Screenshots

https://share.getcloudapp.com/d5uNejlY

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		const afterBreadcrumb = {
+			'breadcrumb-position': 'astra_header_after',
+			'breadcrumb-disable-home-page': 0,
+		};
+		await setCustomize( afterBreadcrumb );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-breadcrumbs-wrapper' );
+		const enablehomePage = await page.$eval( '.ast-breadcrumbs-wrapper', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js:14]: const declare as camelCase for enablehomePage
```suggestion
		const enableHomePage = await page.$eval( '.ast-breadcrumbs-wrapper', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js]: Done changes

File: tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+		await publishPost();
+		await page.goto( createURL( '/single-page' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-breadcrumbs-wrapper' );
+		const disablePage = await page.$eval( '.ast-breadcrumbs-wrapper', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( disablePage ).toBeNull( );
+	} );
+
+	it( 'enable breadcrumb on single ppost for inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js:85]:
```suggestion
	it( 'enable breadcrumb on single post for inside header position should apply corectly for inside position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+		await setCustomize( afterBreadcrumb );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-breadcrumbs-wrapper' );
+		const enableHomePage = await page.$eval( '.ast-breadcrumbs-wrapper', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( enableHomePage ).toBeNull( );
+	} );
+
+	it( 'enable breadcrumb on blog page for inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js:18]:
```suggestion
	it( 'should enable breadcrumb on blog page for inside header position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js]: Modified

File: tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js (hunk 3 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js]: Modified

**Review by bsfsnehal — APPROVED** [review]

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**